recipe/dashboard/recipeimplementation.go

```diff
@@ -0,0 +1,51 @@
+/* Copyright (c) 2022, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package dashboard
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/supertokens/supertokens-golang/recipe/dashboard/dashboardmodels"
+	"github.com/supertokens/supertokens-golang/supertokens"
+)
+
+func makeRecipeImplementation(querier supertokens.Querier, config dashboardmodels.TypeNormalisedInput, appInfo supertokens.NormalisedAppinfo) dashboardmodels.RecipeInterface {
```

Comment:
- querier not needed for this recipe. 
- config not needed to be passed here - it should be passed in the shouldAllowAccess function arg (see node PR)
- appIinfo not needed

Reply:
Done